Podcast/EpisodeSubjectView.swift

```diff
@@ -173,7 +173,19 @@ class EpisodeSubjectView: UIView {
     func setupWithEpisode(episode: Episode) {
         episodeNameLabel.text = episode.title
         dateTimeLabel.text = episode.dateTimeLabelString
-        descriptionLabel.attributedText = episode.attributedDescription
+        // this is to avoid newlines/paragraphs showing up after truncating text
+        let stringWithoutNewlines = episode.attributedDescription.string.replacingOccurrences(of: "\n", with: "")
```

Comment:
is this the same code as above? If so, maybe include this as an extension to NSMutableString as a function that takes in an episode

Reply:
done! i made an extension to `NSAttributedString`